### PR TITLE
Private Room enhancement

### PR DESCRIPTION
**Pull request by hawkeye-e**
- Allow freely summon non bad-friend characters into private room. 
- Allow MMF and FFM options regardless of job in private room.

### CODE REVIEW

**Comment by bogus-things** [comment]
this is awesome! have you tried to make it work outside private rooms?

**Comment by hawkeye-e** [comment]
at the beginning i try to figure out how mmf situation work in job map but the mmf command by male character never show up
i think illusion hard coded mmf command to show up only if the character is in the bad friend action point

**Comment by bogus-things** [comment]
yeah I was seeing something similar when I tried to make MMF work in job maps. kind of a bummer they designed it like that

File: README.md [hunk 1]
```diff
@@ -1,46 +1,12 @@
-# RG_ActionPatches
```
Comment (bogus-things) [inline README.md]: could you remove this file from your PR? I don't want it to overwrite the main one

File: src/RGActionPatches.csproj [hunk 2]
```diff
@@ -32,49 +32,49 @@
   </PropertyGroup>
   <ItemGroup>
     <Reference Include="Assembly-CSharp">
-      <HintPath>C:\Games\RoomGirl\BepInEx\unhollowed\Assembly-CSharp.dll</HintPath>
```
Comment (bogus-things) [inline src/RGActionPatches.csproj]: can you move the reference changes out of this file, too? if you make a `RGActionPatches.csproj.user` you can put them there and they'll override the properties in this file. or you can remove this file from the PR and I'll manually add the compile tags after merging

File: src/RGActionPatchesPlugin.cs [hunk 3]
```diff
@@ -12,7 +12,7 @@ public class RGActionPatchesPlugin : BasePlugin
     {
         public const string PluginName = "RG Action Patches";
         public const string GUID = "com.bogus.RGActionPatches";
-        public const string Version = "0.2.0";
+        public const string Version = "0.2.0.1";
```
Comment (bogus-things) [inline src/RGActionPatchesPlugin.cs]: you can take this one out, too, and I'll update it post-merge

**Review by bogus-things — CHANGES_REQUESTED** [review]
looks good! if you can remove the few files I commented on from the PR to avoid conflicts, I'll go ahead and merge these changes in

**Comment by hawkeye-e** [comment]
all done!

**Review by bogus-things — APPROVED** [review]
Great stuff! Thanks again for putting this PR up

**Pull request merged**